go.mod

```diff
@@ -92,6 +92,7 @@ require (
 	github.com/golang/protobuf v1.5.3 // indirect
 	github.com/google/btree v1.0.1 // indirect
 	github.com/google/gnostic v0.6.9 // indirect
+	github.com/google/gnostic-models v0.6.8 // indirect
```

Comment:
Denne transitive modulen her skaper kompileringsfeil:
```
# k8s.io/client-go/applyconfigurations/meta/v1/go/pkg/mod/k8s.io/client-go@v0.27.3/applyconfigurations/meta/v1/unstructured.go:64:38: cannot use doc (variable of type 
*"github.com/google/gnostic/openapiv2".Document) as 
*"github.com/google/gnostic-models/openapiv2".Document value 
in argument to proto.NewOpenAPIData
```

Reply:
Det virker som at `client-go` må bumpes samtidig ref https://github.com/kubernetes/kubernetes/issues/118340
og at `gnostic` utgår til fordel for `gnostic-models` ref https://github.com/kubernetes/kube-openapi/pull/402

Reply:
Tror vi får vente på oppdaterte dependencies overalt